index.ts

```diff
@@ -0,0 +1,133 @@
+import express from "express";
+import { MongoClient, Db, Collection, ObjectId, WriteOpResult } from "mongodb";
+import * as apiconfig from "./apiConfig.json";
+import * as access from "./access.json";
+import moment = require("moment");
+import assert = require("assert");
+import { NextFunction } from "express-serve-static-core";
+import bodyParser = require("body-parser");
+
+const app = express();
+const client = new MongoClient(apiconfig.mongoUrl, { useNewUrlParser: true });
+
+let db: Db;
+
+client.connect((err) => {
+    assert.equal(null, err);
+    db = client.db("VaorraJS");
+});
+
+const authentication = (req: express.Request, res: express.Response, next: NextFunction) => {
+    let user = req.headers["x-access-user"];
+    let token = req.headers["x-access-token"];
+
+    user = user ? user.toString() : undefined;
```

Comment:
Send responsestatus in user variable definition [Line 24/25]

Reply:
Rather not do that, we'd need returns then. I'll just .toString() in the third if statements.